### DISCUSSION TITLE
blog/ai-jurisprudence-academic-bubble

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#4] [Announcements]
# blog/ai-jurisprudence-academic-bubble

多少年过去了，整个人工智能法学圈，仍然没有摆脱这种虚假的反事实假设和推理。在这个假设还停留在反事实假设的阶段，一切关于它的推理，要么是虚假的，不可判定真假的，要么和除去人工智能障眼法的普通法律推理没有任何差别。

https://rercel.com/blog/ai-jurisprudence-academic-bubble

**Comment by ghost** [comment]
## 前言

这里是一位从google搜索而来的读者，非常高兴能够阅读这篇blog，人工智能在伦理和法学上确实是一个不可逃避的问题。

## 观点

1. 用传统经验主义，正如波普尔批判的那样，人们依赖于类似数学归纳法的方法论，从而推导出某种“历史规律”，被称之为“历史决定论”。

2. 用计量经济学的角度来看，“假设，我变成了天使，你会不会接受我？”可以看作是一个logistic回归（二元分类），如果依赖于经验性的样本，拟合之后只是概率事件的模型。

3. 用金融市场一种更为现实的例子，当某对冲公司破产的案例是大于5个标准差，也就是概率小于10^-n，但这是根据过去有限样本得出的测试差，会有不少人感到意外；另一个例子就是耳熟的黑天鹅和“已知的未知”，经济学经常难以界限“过拟合”。

4. 在现实市场，人们也会去假设一些“市场效率”、“信用违约风险”甚至是对未来利率的预测。

5. 说回哲学一些观念，**语言本身是一种隐喻**，“现实”是各种符号再次拼凑而成，这也是现代媒体的艺术，人工智能就像是“符号”^2，信息不完备性+不对称，后真相，还有文章提及的反事实。

我觉得AI除了本身伦理问题，更多反讽了人类作为主体在现代化媒体、市场、法律的一些问题。

--- 

上边有些疑问我觉得我可以尝试回答：

> 当 AI 作为个人或组织的财产是一个异常清晰的法律事实时，讨论赋予 AI 以著作权的意义什么？在讨论这些问题之前， 是否应当重新检讨两个问题：AI 在何种意义上值得被称为人，以及，是否法律的主体不限于自然人或法人那样的“人”？

目前的AI仍然是拟合+模仿人类的话语，应该追溯该模型使用的样本，即样本的商用使用权，但AI本身只会加剧信息不完备从而导致“人为制造的风险”，而这种风险往往是系统化的，如果样本中有某种未经察觉的错误而被重复训练和使用，是模型的样本出了问题，但却影响了可能付费使用者，也就是这种风险的不对称性。


## 最后

很可惜的是现在并没有迎来公共政策的“机会之窗”，人们正在科技泡沫中醉生梦死，我身边那些连openai的api都没接上甚至没使用过的，却给我说这种技术的应用，相比人工智能法学，人可能问题更大，大多数人也是根据各种符号拼凑而成的信息去认知世界。

[槐序](https://www.xushilu.com/)
11 April 2023

**Reply by Pathsis** [reply]
谢谢评论！😊

先说一下文章的两个要点：第一，法律基于对人和社会很基本的假设，只要AI还不足以是人，或者人的概念还没有实质改变，无论AI的能力发展到何种程度，法律目前的框架和范式都为其保留了足够的空间；第二，目前的人工智能法学，依赖一个反事实假设，所做的推理是无法判定的，假设的轻而易举，使研究者误以为所做的真理是真实的。

这篇文章不涉及AI的具体实现和效果，也不涉及AI自身的推理。

这篇文章还可以说以人工智能法学为例，批评那些没有注意到AI还不足以威胁本领域基本假设的虚假人工智能相关研究。

**Comment by wang2mazi** [comment]
人工智能法学有些是讨论长远的未来，但也有些是在讨论当前或者可想象的未来的问题。
例如，甲在使用乙制造的弱人工智能的过程中损害了丙，如何判断乙在何种情况下需要承担产品缺陷责任？
再如，甲将乙创作的作品喂给某个弱人工智能，丙运用这个弱人工智能创作出一些作品，甲、丙在何种情况下需要乙之许可、在何种情况下需要向乙支付报酬？如果不需要许可，但需要支付报酬，则报酬金额如何确定？

**Reply by wang2mazi** [reply]
另外，“目前的人工智能法学，依赖一个反事实假设，所做的推理是无法判定的”，这一观点恐怕涉及到法学之性质的极大误解。法学研究的是规范，而不是经验事实。所有的法学研究都依赖于某个反事实假设，但这些研究之对错依然是可判定。例如，罗翔的许多观点都依赖于他对于“张三”这个虚构人物的想象。我们不能说，因为张三这个人事实上不存在，所以罗翔所做的推理是无法判定的。法学研究的可检验性可能是个复杂的话题，但几乎所有人都承认的一点是，许多法律命题是这样的结构：如果出现了事实F，则法院应当判决D。“如果出现了事实F，则法院应当判决D”这个命题的可检验性决不依赖于事实F实际上是否出现。

**Reply by Pathsis** [reply]
感谢评论！

1. 完全赞同你的如下看法：“人工智能法学有些是讨论长远的未来，但也有些是在讨论当前或者可想象的未来的问题。”你所提的这两个问题，都是真实、有价值的法学问题。
2. 不是很认同你的如下看法，即你认为我误解了法学的性质。法学研究的不只是规范，而还有经验事实。实际上，法学根本上应被视为一种经验性而非规范性的学科：法律基于我们的一般社会现实，它旨在对其做出规范和控制，若事实是x，则应当y，这里，x必须是普遍的事实。法学不会处理大规模的非事实、反事实，比如，假如外星人加入了地球，成为这个星球的“新人类”，那么应当赋予他们人的权利。外星人加入了地球，不是一个普遍的事实——它压根儿就不是事实，它是一个反事实。这个问题会成为科幻小说、艺术甚至哲学的主题，但还不是法律和法学的主题，直到有经验证据表明，这是或将是一个普遍的事实。
3. 我非常抱歉，我对**反事实**这一令人困惑和容易引起望文生义的术语缺乏基本的界定。

- 一个条件句的一般形式是，**若x，则y**。如果x被假定是与事实相反的，则这个条件句一就是一个[反事实条件句](https://plato.stanford.edu/entries/counterfactuals/#:~:text=In%20philosophy%20and%20related%20fields%2C%20counterfactuals%20are%20taken%20to%20be%20sentences%20like%3A)。例如，“**如果荆轲成功刺杀了秦王，那么秦国不会统一六国**”，是一个典型的反事实条件句。这语句预设了，**荆轲成功刺杀秦王**，不是一个事实。

- 反事实条件句经常会与虚拟条件句混同，或者在许多的文本中经常被交替使用。一个典型的虚拟语气是，“**假如我变成了百万富翁，那么我就能赢得我意中人的青睐**”，这里假设一个面向未来的事实，假设它将来发生，并讨论在拥有这个事实的将来世界中，会发生的事情（“我赢得我意中人的青睐”）。

- 虚拟条件句与反事实条件句有什么不同呢？首先，从形而上学的角度说（也就是从以下角度，若某项被假定本来不存在于我们的世界w的事实x发生，那么拥有x的可能世界w'会发生什么事实，比如，会发生y事实），我认为，基本上它们的差别很小，可以互相交换使用。这样，“假如荆轲成功刺杀了秦王，那么秦国不会统一六国”也是一个虚拟条件句。其次，从语义学的角度说，它们之间有很大的差别，反事实条件句的前件x的指称是不存在的。这样，一个更准确的反事实条件句实例是：“假如人类是更高智慧生命操控的NPC（x），那么我们的宇宙可能只是众多宇宙中的一个（y）”，在这里，x没有指称，因为在经验世界中，我们无法为x建立一个限定摹状词，也就是说，在经验世界中没有一个ø，使得x是真的。由于这个缘故，我们并不能像一般的虚拟条件句那样开展可能世界的模态推理，因为缺乏拥有x事实的可能世界的资料，没有办法建立以它为前件的条件句的推理。

- 可以这样简单理解这个问题。一个最简单的例子是，假如外星生命存在，那么我们的星球、星系乃至宇宙的命运会是如何的。我们只能依靠想象或者幻想。假设一个可能世界w，包含x事实（外星生命存在），但是，仅仅是想象而已，没有办法做任何实质性的推理。也正因为这个缘故，我们关于x的想象，都是我们对自身的认识的投射，因此实际上，有关外星人的推理，实际上与没有外星人的推理，没有区别。类似的，强人工智能（我假设它拥有起码的意识或人格）假如出现，那么我们的世界会怎么样，是一个根本无法开展实质推理的问题。

4. 按照上面的界定，说法学研究的都是反事实假设，是错误的。截至此刻，张三这个人没有杀人。现在，”假设他曾杀了人，那么他应当被法律处以死刑“。这是一个虚拟条件句，但严格说，不是反事实条件句。从形而上学上说，在我们的世界w中，确实不存在x事实（张三杀人）。但我们可以假设一个世界w'中包含x事实，并且推断，w'会发生其他什么事实（他会被处以死刑）。从w到w‘的假设不是纯粹的想象，而是有着大量的经验材料作为根据；重要的是认识到，x的指称物是存在的，是可以确定的。

5. 在罗翔的例子中，张三这个人被假定是虚构的。这里的逻辑是，”张三“这个名称被假定**没有**任何指示物，而不是所设想的指示物本身**实际**不存在。”假如张三杀了人， 那么他应当被处以死刑“，与”假如人类制造了有心灵的机器，那么法律应当赋予它们人格“，按照我的界定，是完全不同的条件句。前者涉及一个非事实，后者涉及一个反事实。在前一个条件句中，如果需要，我们能立即在世界中找到一个实例，并分配给”张三“这个名称，使得前件是真的。这就是为什么罗翔的例子所涉及的推理都是真实的。但是，在后一个条件中，”有心灵的机器“这个谓词被假定**有**指称物，然而这个指称物在我们的真实世界w中从截至目前而言的经验而言上不存在。这正是“反事实“这个词的真正意义所在。

6. 这样，可以评论你的这样一个观点：”许多法律命题是这样的结构：如果出现了事实F，则法院应当判决D“。很明显，如果F是一语义学上的反事实，那么这个条件句就是不可判定的。

7. 我希望我上面的论述大致说清楚了形而上学反事实和语义学反事实的区别。我的文章所说的反事实是语义学上的。

8. 我希望强调下我的文章的两个要点：（1）法学基于对人和社会的特别基本的假设，这个假设为AI的能力发展理由了足够的空间，只有当其发展触及了这些假设，一种真正的人工智能法学才是必要的；（2）假如我们制造了能够向我们提起公民权利诉求的机器，假如我们制造了有心灵或意识的机器，法律应当如何对待它，抱歉，在这个事实出现之前，我们所做的推理，只是将有关我们自己的认识投射到一个想象的x之上，因此这样的所有推理，要么是不可判定的，要么与不做这样假设的推理没有差别。

9. 最后，我非常认同一点，除了以上两点之外，人工智能法学的相关研究是富有成效和有价值的。我意外地发现，[许多读者正确指出我的文章在这个问题上有失公允](https://mp.weixin.qq.com/s/QWNnvOaao0IsYwJUKEqWDg)。我的文章的意义在于，从一种基本的前提出发，提醒某些人工智能法学研究者，不要基于“假如人类造出了有（某种程度的）心灵或意识的人，那么法律应当如何对对它们”的假设来设定议题。必须承认，大量的人工智能法学研究基于这个假设来设定议题。这种假设在那些对人工智能法学只有一点外行兴趣的人群中，更是泛滥。我开头提出的那个例子，就是一个证据。

**Reply by wang2mazi** [reply]
看来我误解了你之前所说的“反事实条件句”的意思。确实，我们不能说所有的法学研究都依赖于反事实条件句。但这是否意味着所有的法学研究都不可以依赖于反事实条件句呢？

1. 我猜你可能想表达这个意思：假如查尔斯死了，而且他一生中从来没有遇到过任何危险，那么“查尔斯是不是勇敢的”这个问题的答案是不可知的。

不过，恐怕这一观点混淆了经验与规范。你所举的例子中，关于反事实条件句的推理是这种结构：“如果出现了事实F，则会出现事实G。”但法学命题不是这种结构。法学命题的结构是：“如果出现了事实F，则法院应当判决D”。这里的“应当”表明所研究的对象的是规范，而不是事实。法学命题谈论的是原则与规则，而不是因果关系。即使事实F不可能存在，这一法学命题依然是可判定的。判定的方法便是依赖更抽象的原则或规则。法学家需要寻找一个这样的更抽象的规范：“如果出现了有属性N的事实，则法院应当判决D。”接着便可根据法律推理来探讨“如果出现了事实F，则法院是否应当判决D”的问题。这一探讨不依赖于事实F是否在语义上可能出现。法学家可以得出结论说：“如果出现了事实F且事实F有属性N，则法院应当判决D；如果出现了事实F且事实F没有属性N，则法院不能根据这一理由判决D。”这一结论是否正确，是可以检验的。检验的方法便是探讨“如果出现了有属性N的事实，则法院应当判决D”这一更抽象的规范是否有法律效力。

2. “张三杀人”中的“张三”可以立即找到一个实例，因为杀人是个非常常见的事实。在这种情况下，确实，“张三杀人”只是一个虚拟条件句，而不是一个反事实条件句。但是，法学家也谈论过很多稀奇古怪的例子。这些例子在现实中可能存在，也可能不存在。这些例子可能只在考试试卷或法学期刊上出现。但当考官在试卷上出题的时候，考官没有义务在出题前就保证这些例子在现实中出现了。这些考题的正确答案依然是可以判断的。判断的方法便是依赖于更抽象的规范。考官出这些怪题的目的不是要为难学生，而是让学生通过这些稀奇古怪的例子来掌握抽象的法律规范。换言之，法学讨论中完全可以运用语义学反事实假设，以便更好地说明一些法律命题之含义，更好地解释那些抽象的法律规范应当如何适用。

3. 除了法学领域之外，语义学反事实假设也在其他一些领域发挥作用。例如，普特南用孪生地球的思想实验来说明自然类的性质。实际上不存在孪生地球。“孪生地球”这个词没有指称。在经验世界中，我们无法为“孪生地球”建立一个限定摹状词。也就是说，在经验世界中没有一个ø，使得“孪生地球”存在。但是，我们依然可以讨论以孪生地球存在为前件的条件句的推理。

4. 对于“假如外星生命存在，那么我们的星球的命运会是如何的”这个问题，我猜你想说的是，“假如外星生命存在，那么我们的星球的命运会是如何的”这个问题的唯一探讨方式是“假设一个可能世界w，包含x事实（外星生命存在）”。你大概想说的是，由于缺乏可能世界w的资料，我们不清楚“外星生命存在”这一命题的涵义。你大概认为，我们不知道也没有办法知道“如果外星生命存在，那么这些外星生命是什么样的存在”。

不过，这一观点大概混淆了涵义和指称。“外星生命”这个词项的指称不存在不等于这一词项的涵义不确定。如果一个学者只说“假如外星生命存在，那么我们的星球的命运会是如何的”，而不说清楚这个句子中的“外星生命”这个词项的涵义，那么他的主张确实是不可判定的。但如果他清楚定义了“外星生命”这个词项，他的主张就可以判定。例如，假设他说“假如H. G. 威尔斯的《星球大战》中描述的外星生命存在，那么我们的星球的命运会是如何的”，那么他的这一主张确实是可以判定的。换言之，只要反事实条件句的含义是清楚的，我们就可以做许多实质性的推理，而且这些推理是否有效都是可以判定的。你可以说这是一种想象或幻想，但没有理由说这种实质性的推理是不可检验的。“强人工智能（假设它拥有那些学者们所描述的那种程度的意识或人格）假如出现，那么我们的世界会怎么样？”这个问题可能很难回答，但你不可能在哲学上说关于这个问题的推理是不可判定的。“如果出现了事实F，则会出现事实G”这个命题并非不可检验，只要“事实F”的涵义是清楚的。

5. 我猜你真正想表达的，不是“依赖反事实条件句的人工智能法学不可判定”，而是“依赖于反事实条件句的强人工智能法学没有研究价值”。这就好比：“假如《星球大战》中描述的外星生命存在，那么我们的星球的命运会是如何的”之类的命题是可判定的；不过，关于这种命题之真假的探讨没有什么实际意义。这是因为，“《星球大战》中描述的外星生命存在”存在的概率要么为0要么接近于0。所以，关于这类问题的讨论，可以当作娱乐消遣的游戏，但不值得当作严肃的学术研究。

然而，在我看来，还不能这么快下结论。“强人工智能（假设它拥有那些学者们所描述的那种程度的意识或人格）假如出现，那么我们的世界会怎么样？”这个问题是否有研究价值，似乎取决于两个因素：（1）强人工智能出现的概率；（2）假如强人工智能出现则会造成的社会影响。在我看来，如果强人工智能在未来二十年内出现的概率高于千分之一，那么社会就有必要投入一部分资源用于相关的人工智能法学研究。（当然，没有必要投入过多资源进入这类研究。至于什么程度算“过多”，什么程度算“不及”，我这里说不太清楚。）这类似于帕斯卡的赌局。假如某个事实可能造成的影响极大，那么即使该事实出现的概率极低，社会可能也有理由去考虑假如该事实出现则会如何。正所谓，人无远虑，必有近忧。

**Reply by Pathsis** [reply]
非常感谢赐教！

1. “法学命题”的结构或性质

   你的查尔斯的例子，不能表达我的意思。一般而言，“查尔斯”是一个人的名字，我们都很清楚这意味着什么。因此查尔斯的例子和张三的例子没有什么不同。即使某个条件句中“查尔斯”没有指称，即使存在的查尔斯很早就死了，有关他勇敢与否的推理，在我看来，不是反事实性的。因为，只要我们愿意，随时在我们的世界中找到一个实例，分配给“查尔斯”，使得“若查尔斯如何如何，则应当或会如何如何”的条件句的前件是真的。

   但是，对于“有心灵的机器”或“外星人”这样的概念，若它们出现在一个条件句中，就会使这个条件句变成一个反事实条件句，因为我们不能在我们的世界中找到一个实例，分配给它们，使得诸如“假如人类造了有心灵的机器，则法律应当如何如何对待它们”这样的条件句的前件为真。

   正是由于反事实条件句的前件的这一特征，使得“若出现了事实F，则法院应当判决D”的法律推理不可判定的。

   抛开其他的，我们都同意，法律处理的是我们世界的一般和普遍的事实。法律当然是一种规范，它假设，若世界有事实F，则法律要它变得G。当我们用命令来理解这一点，会变得很容易，无论事实是什么，法律使得它如何如何。但是，这个规范，要是有意义的，前提是，我们确实知道，事实F到底指什么。当“事实F”其实是一个反事实，法律的目标——规制一个事实——就会失去焦点，因为没有靶子供它射击。

2. 法律的性质

   我的法学研究使我相信，一般而言，法律的存在和内容取决于社会事实。前面，我同意你，一般人们会认为，法律命题的一般结构是规范性的，即对于事实F，应当G。但看起来我认为，法律命题的一般结构是事实性的，即对于事实F，会发生事实G。我认为，这是一个复杂的问题，尽管我必须说，的确如此。

   然而，即使不是这样，我关于法律的反事实推理的观点，也是成立的，正如我前面所阐明的那样。如果你愿意和我继续探讨这个深刻的话题，非常欢迎。

3. 法学家稀奇古怪的例子

   我想，在司法考试上应该没有出现类似“假如某官员使用的有心灵的机器隐瞒领导贪污了公款”或“假如张三被外星人杀害”之类的问题，所真实出现的那些奇怪的题目都在我们世界的一般和普遍现实范围之内的。如果出现上述类似问题，那么必须被视为反事实的，是不可判定的，不是真实的法律问题。假如人们对之做出了推理，尽管十分真诚，正如我在正文所说的， 是虚假的。

4. 哲学问题

   正如你所说，“孪生地球”的例子出现在严肃的哲学文本中。当然，抛开普特南的具体结论和用意，就我们的主题而言，这不意味着类似的问题是真实的法学问题。我在正文已经表达了这一意思。

5. 内涵和指称

   我不认为一个找不到任何指称物（实例）的词项有真实的内涵。当然，这是一个值得争议的哲学问题。但，这不是一个值得争议的法学问题，它在法学中可以被轻而易举打发。法律和法学，不会处理找不到任何指称物的概念或词项。重申一遍，法律和法学处理我们世界的普遍事实，它们不处理幻想和想象。

6. 强人工智能假如出现

   这又回到了问题的起点。这个问题，是一个技术问题、公共政策问题、哲学问题乃至伦理学问题，但还不是一个真实的法学问题。

7. 人无远虑，必有近忧

   我绝不是完全否认人工智能法学研究。我的目标是为了提醒，基于反事实设定法学研究议题的虚假之处，这个现象无论是在学界还是一般公众那里，都是普遍的。

欢迎继续探讨☺️

**Reply by wang2mazi** [reply]
总结一下，可不可以这么理解，兄台以为，“以反事实条件句为前件的推理”必然为假或者是不可判定的。
我认为这个观点殊属费解。
例如：
“如果存在能够故意杀人的人工智能，则存在能够故意杀人的人工智能。”
这个推理是否有效，有何不可判定？
又如：
前提：如果理性存在者故意杀人，则该理性存在者应当受到法院惩罚。
结论：如果人工智能A是一个理性存在者，则“如果A故意杀人，则A应当受到法院惩罚”。
这个推理是否有效，有何不可判定？
再如：
前提之一：如果二十年后的聊天机器人写的小说与人类小说家的小说有竞争关系，则二十年后的聊天机器人的制造者或使用者应当向人类小说家支付报酬。
前提之二：如果二十年后的聊天机器人写的小说与人类小说家的小说有替代关系，则果二十年后的聊天机器人写的小说与人类小说家的小说有竞争关系。
结论：如果二十年后的聊天机器人写的小说与人类小说家的小说有替代关系，则二十年后的聊天机器人的制造者或使用者应当向人类小说家支付报酬。
这个推理是否有效，有何不可判定？

**Reply by wang2mazi** [reply]
“法律和法学，不会处理找不到任何指称物的概念或词项。”
这显然是一个错误观点。
如果（1）“著作权法的宗旨是增加消费者剩余”，则（2）“如果2025年的聊天机器人写长篇小说可以增加消费者剩余，则2025年的聊天机器人写长篇小说受著作权法保护，除非著作权法修改了。”
这一推理不是法律推理吗？这一推理不是有效的吗？
“可以写长篇小说的聊天机器人”就是一个找不到任何指称物的概念或词项。
显然，法律和法学不仅处理能够找到指称物的概念或词项，也处理找不到任何指称物的概念或词项。
这是因为，人们要依赖对于法律的合理预期来规划自己的生活，而人类生活的一个重要方面就是发明和创造之前不存在的新事物。
人类历史就是幻想和想象变为真实的历史。
重申一遍，法律和法学处理我们世界的普遍事实，包括幻想和想象。

**Reply by Pathsis** [reply]
关于那个推理属否有效的问题，我没有前面之外的话。我想这是一个哲学和逻辑学的问题，而不局限于法学，不是法学能够独立解决的问题。

我也重申一遍，如你所言，法学处理事实，但是，不处理幻想和想象，因为，幻想和想象，不是事实。

**Comment by wang2mazi** [comment]
如果兄台所言为真，那么，所有的法学研究，也就全都成了学术泡沫了。
如果“蔡徐坤于2026年2月2日故意杀死蔡徐坤的次子，且不存在正当防卫等法定免责事由，则蔡徐坤的这一行为构成故意杀人罪。”
这句话中，“蔡徐坤的次子”就是一个不存在指称物的词项，但该句子有适真性，并且该句子的真值可以通过法律推理来判定。
如果这个句子的真值都不可判定，那人就无法依赖法律或法学来规划自己未来的行为，法律也就不能起到指引人未来的行为的作用。
如果人工智能法学的真值都不可能判定，那么科研人员就无法依赖法律或法学来规划关于人工智能的发明创造。
